### PR TITLE
frequency once for funding, UI changes in Project name and description

### PR DESCRIPTION
**Pull request by tejassonar**
- Frequency once for funding
- Project name, not auto capitalized
- Project description changed for all funding projects

### CODE REVIEW

**Comment by norbertschuler** [comment]
- [ ] Please rebase this branch onto develop (or merge if you cannot rebase) and solve conflicts.

**Comment by norbertschuler** [comment]
- [ ] The forbidden image https://cdn.plant-for-the-planet.org/media/cache/project/large/61b3479b0c964337676750.jpg
 is hopefully not also used on production.

**Comment by norbertschuler** [comment]
- [ ] I understood @sagararyal the way this text about tax should be replaced by the project description:

image

image

**Comment by norbertschuler** [comment]
- [ ] I tried doing a funding donation "once", but in the end the donation request repeated for a long time and I always got the `pending` state from the staging backend and also the `bank` field is still showing empty in this pull request.

Bildschirmfoto 2021-12-12 um 10 01 23

**Comment by sagararyal** [comment]
This can be optimized.
Screen Shot 2021-12-12 at 1 10 23 PM

File: public/locales/de/common.json [hunk 1]
```diff
@@ -133,5 +133,6 @@
   "beneficiary": "Kontoinhaber",
   "copied": "Kopiert",
   "clickToCopy": "Zum Kopieren anklicken",
-  "bank": "Bank"
+  "bank": "Bank",
+  "fundingAndBouquetProjectDescription": "Mache steuerlich absetzbare Spenden für über 160 Renaturierungs- und Schutzprojekte. Deine Reise zu einer Billion Bäume beginnt hier."
```
Comment (sagararyal) [inline public/locales/de/common.json:137]: use this as `fallbackdescription`

File: src/Donations/Micros/FrequencyOptions.tsx [hunk 2]
```diff
@@ -2,16 +2,18 @@ import { useTranslation } from "next-i18next";
...
-  let customfrequencies = ["once", "monthly", "yearly"];
-  if (paymentSetup.purpose === "funds") {
-    customfrequencies = ["monthly", "yearly"];
-  }
+
+  const customfrequencies = paymentSetup.frequencies;
+  // let customfrequencies = ["once", "monthly", "yearly"];
+  // if (paymentSetup.purpose === "funds") {
+  //   customfrequencies = ["monthly", "yearly"];
+  // }
```
Comment (sagararyal) [inline src/Donations/Micros/FrequencyOptions.tsx:16]: cleanup

File: styles/common.scss [hunk 3]
```diff
@@ -104,7 +104,7 @@
 .title-text {
   font-size: 24px;
   font-weight: 800;
-  text-transform: uppercase;
+  // text-transform: uppercase;
```
Comment (sagararyal) [inline styles/common.scss:107]: clean

**Comment by tejassonar** [comment]
> This can be optimized. Screen Shot 2021-12-12 at 1 10 23 PM

I have used the same Logic.
image

**Pull request merged**